### DISCUSSION TITLE
Default system prompt for all users

### DISCUSSION THREAD

**Discussion by rtomik** [open-webui/open-webui#3422] [Q&A] [answered]
Hello,

is there a way to define default webui system prompt for all users that will login to the app? 

Thanks

![image](https://github.com/open-webui/open-webui/assets/31215293/347a0d0b-0f8c-4c65-9557-9cf5f77c737c)

**Comment by AndyEverything** [comment]
I am also looking for this. It would be great if you could set a default system prompt, a default language and possibly a default theme, which I can preset as admin.

**Comment by clang88** [comment, marked as answer]
Suggestion for now: Why not create a custom model with the prompt set that you want and limit the users to that model?

**Reply by ashley-graves** [reply]
Issue: this will not support dynamic prompts (USER_NAME, USER_LOCATION, CURRENT_DATE, etc)
![image](https://github.com/user-attachments/assets/1d29ec85-f2c0-4352-b505-c4f7aa6ccc5f)
